Image refresh part 7 (#7752)
fishtown-analytics/docs.getdbt.com: website/docs/guides/sl-snowflake-qs.md

1. Create a new project in <Constant name="cloud" />. Navigate to **Account settings** (by clicking on your account name in the left side menu), and click **+ New Project**.
2. Enter a project name and click **Continue**.

> 3. In the **Configure your development environment** section, click the **Connection** dropdown menu and select **Add new connection**.

Reviewer: Same feedback as earlier
```suggestion
3. In the **Configure your development environment** section, click the **Connection** dropdown menu and select **Add new connection**. This directs you to the connection configuration settings. 
```

>   This directs you to the **Add new connection** page. 

Reviewer: Same feedback as in the earlier doc, 
```suggestion
```

    <Lightbox src="/img/snowflake_tutorial/dbt_cloud_snowflake_development_credentials.png" title="dbt - Snowflake Development Credentials" />

11. Click **Test connection**. This verifies that <Constant name="cloud" /> can access your Snowflake account.

> 12. If the test succeeded, you can set up a repository. If it failed, you may need to check your Snowflake settings and credentials.

Reviewer: ```suggestion
12. If the test succeeded, click **Save** to complete the configuration. If it failed, you may need to check your Snowflake settings and credentials.
```
